### ISSUE TITLE
Time Picker Invalid Date Firefox

### ISSUE THREAD

**Issue by taliesinmillhouse** [chronotruck/vue-ctk-date-time-picker#60]
Hi,

When using the time picker in Firefox, selecting a time returns invalid date. Works fine in Chrome. Possibly something to do with [this](https://stackoverflow.com/questions/3257460/new-date-is-working-in-chrome-but-not-firefox)?

The code I've written.
```html
<vue-ctk-date-time-picker v-model="start_time"
formatted="h:mm a" format="h:mm a" time-format="h:mm a"
label="Start Time" :minute-interval="5" disable-date/>

var time_section = {
        data: function() {
            return {
                start_time: null,
            }
        },
}

```
Some screenshots.
![](https://user-images.githubusercontent.com/30204322/48521635-7d1e2c00-e8c9-11e8-9a6b-035bdacf7d4e.png)
![](https://user-images.githubusercontent.com/30204322/48521637-7e4f5900-e8c9-11e8-87cb-107238339d61.png)

**Comment by LouisMazel** [comment]
Hi, 

All is fine for me on Firefox (MacOS & Window)
I tested with `null` like you and nothing

I don't know what's the problem for you...

**Comment by taliesinmillhouse** [comment]
So I've updated to the latest release, v1.3.6. Tried it on Firefox on both Ubuntu 18.10 and Windows as well Chrome and Safari on IOS, all producing the same result.

**Comment by taliesinmillhouse** [comment]
Figured out the cause to the problem.

On line 208 of [vue-ctk-date-time-picker.vue](https://github.com/chronotruck/vue-ctk-date-time-picker/blob/master/src/vue-ctk-date-time-picker/vue-ctk-date-time-picker.vue) the moment object being instantated isn't given a time format. Because no format is given moment, it assumes ISO standard.

So if a non ISO 8601 standard time format is given to the format property on the `<vue-ctk-date-time-picker></vue-ctk-date-time-picker>` it causes the invalid date error. See the link on the [documentation](http://momentjs.com/docs/#/parsing/string/).

**Comment by anshizzle** [comment]
@LouisMazel while I work on a patch - can we reopen this issue; it's clearly reproducible.

**Comment by anshizzle** [comment]
@taliesinmillhouse here's a working fork. https://github.com/anshizzle/vue-ctk-date-time-picker

You'll also have to make sure `@babel/runtime` is in your yarn dependencies.

**Comment by taliesinmillhouse** [comment]
Awesome, thanks a lot!

**Comment by LouisMazel** [comment]
Thanks guys ! 

It's fixed on #85

I release this soon !
